assets/css/style.scss

```diff
@@ -167,7 +167,6 @@
 
 // Element spacing.
 .wc-block-grid__product {
-	.wc-block-grid__product-image,
```

Comment:
This seems to affect other blocks. This is Top Rated Products, for example:

`trunk` | This branch
--- | ---
![imatge](https://user-images.githubusercontent.com/3616980/155563342-98e6e5e8-447b-4a1a-8b14-90aafeea9eb6.png) | ![imatge](https://user-images.githubusercontent.com/3616980/155563406-8e6859f7-c5de-4f2f-82c2-8312e196844b.png)

Reply:
Just fixed in [this way](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5925/files#diff-cd886530abfa495dc6782189ca779f1304bb73bc1758e5d3f0968448dfb67abeR171). What do you think?